.github/workflows/ci-lint.yml

```diff
@@ -17,28 +17,63 @@ on:
 
 env:
   GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  GOLANGCI_LINT_VERSION: v1.50
 
 jobs:
   golangci:
-    container: golangci/golangci-lint:latest
     name: Lint Go
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
         with:
           ref: ${{ inputs.ref }}
-      - name: Tidy
+      - uses: actions/setup-go@v3
```

Comment:
Let's give this a `name`

Reply:
Ack. Done for the new steps as well as one of the existing steps that was missing it.